packages/table/preview/mutableTable.tsx

```diff
@@ -220,43 +227,19 @@ export class MutableTable extends React.Component<{}, IMutableTableState> {
     // ===============
 
     public render() {
-        const renderMode = this.state.enableBatchRendering ? RenderMode.BATCH : RenderMode.NONE;
-
+        const layoutBoundary = this.state.enableLayoutBoundary;
         return (
             <div className="container">
-                <Table
-                    allowMultipleSelection={this.state.enableMultiSelection}
-                    className={classNames("table", { "is-inline": this.state.showInline })}
-                    enableFocus={this.state.showFocusCell}
-                    fillBodyWithGhostCells={this.state.showGhostCells}
-                    getCellClipboardData={this.getCellValue}
-                    isColumnResizable={this.state.enableColumnResizing}
-                    isColumnReorderable={this.state.enableColumnReordering}
-                    isRowHeaderShown={this.state.showRowHeaders}
-                    isRowReorderable={this.state.enableRowReordering}
-                    isRowResizable={this.state.enableRowResizing}
-                    loadingOptions={this.getEnabledLoadingOptions()}
-                    numFrozenColumns={this.state.numFrozenCols}
-                    numFrozenRows={this.state.numFrozenRows}
-                    numRows={this.state.numRows}
-                    onSelection={this.onSelection}
-                    onCompleteRender={this.onCompleteRender}
-                    onColumnsReordered={this.onColumnsReordered}
-                    onColumnWidthChanged={this.onColumnWidthChanged}
-                    onCopy={this.onCopy}
-                    onFocus={this.onFocus}
-                    onVisibleCellsChange={this.onVisibleCellsChange}
-                    onRowHeightChanged={this.onRowHeightChanged}
-                    onRowsReordered={this.onRowsReordered}
-                    ref={this.refHandlers.table}
-                    renderBodyContextMenu={this.renderBodyContextMenu}
-                    renderMode={renderMode}
-                    renderRowHeader={this.renderRowHeader}
-                    selectionModes={this.getEnabledSelectionModes()}
-                    styledRegionGroups={this.getStyledRegionGroups()}
+                <SlowLayoutStack
```

Comment:
..not that it really matters, and maybe I'm misunderstanding something -- but would it have been better to split the changes to truncated format and the changes to the preview into separate commits?

Reply:
yeah, and might still need to do that, but i'm using this slowlayoutstack to test my changes to truncatedformat.